### ISSUE TITLE
Configurations are different to examples when imported

### ISSUE THREAD

**Issue by Kogoro** [FeatureIDE/FeatureIDE#739]
#### Prerequisitives
* **Git branch/Commit-ID:**  `release3.5`

### Issue description
When importing the examples from #730, #729 and #732, I noticed that configurations of the examples are changed to the smallest configuration through opening the editor. The file seems to be fine nevertheless.
Also after changing the configuration, saving, and opening the configuration again this happens.

**Comment by Subaro** [comment]
@tthuem It is now fixed. The configuration editor does not get any save flags or wrong configuration when opening a configuration.

@skrieter 

After creating a configuration manager, the configuration was additionally set to the specific instance created in the configuration editor. Setting a configuration for a configuration manager set the configuration and the persistentObject but not the persistentObjectSource. Instead of setting the  persistenObject directly I redirected the call to the setPersistentObject method. Could that redirect lead to unwanted behaviours? If not, everything is fine now.
